example/fwup-architecture/dfu.c

```diff
@@ -0,0 +1,59 @@
+#include "dfu.h"
+#include "memory_map.h"
+
+#include <libopencm3/stm32/f4/flash.h>
+
+int dfu_erase_image(image_slot_t slot) {
+    // We just write 0s over the image header
```

Comment:
Maybe: _erase_ -> _invalidate_ ?

Somewhere you'll need a command to do a true sector erase so you can do the firmware update

Reply:
:+1: